packages/slate-editor/src/modules/editor/Editor.tsx

```diff
@@ -206,8 +206,11 @@ export const Editor = forwardRef<EditorRef, EditorProps>((props, forwardedRef) =
 
             const isWithinMenuPortal = isTargetWithin(popperMenuOptions.portalNode?.current);
             const isWithinEditor = isTargetWithin(containerRef.current);
+            const isTextboxElement =
+                clickTarget.tagName.toLowerCase() === 'textarea' ||
+                clickTarget.tagName.toLowerCase() === 'input';
 
-            if (!isWithinEditor && !isWithinMenuPortal) {
+            if (!isWithinEditor && !isWithinMenuPortal && !isTextboxElement) {
```

Comment:
Is it possible to attach event not to `body` but `containerRef` ?

Reply:
Exactly :)